report/search_incidents.go

```diff
@@ -430,37 +72,46 @@ func fetchIncidents(team, ddApiKey, ddAppKey string, since, until time.Time) ([]
 
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Full HTTP response: %v\n", r)
-		return nil, fmt.Errorf("Error when calling `IncidentsApi.SearchIncidents`: %w", err)
+		return nil, fmt.Errorf("Error when searching for incidents: %w", err)
 	}
 
-	if resp.Incidents == nil {
+	if resp.Data.Attributes == nil || resp.Data.Attributes.Incidents == nil {
 		return nil, nil
 	}
 
+	// The raw API response actually contains the incident commander embedded in the incidents, but the SDK doesn't expose it, as this is technically not JSON:API compliant. The SDK only exposes an ID in the relationships.
```

Comment:
The previous implementation was parsing the `commander` field, so it was relying on this technically non-compliant response format: https://github.com/xornivore/incidentist/blob/33efc1adfbc0599a810e5e618c26307ee03449ba/report/search_incidents.go#L170-L171